modules/profilereview/src/Auth/Process/ProfileReview.php

```diff
@@ -286,7 +286,7 @@ protected function redirectToProfileReview(&$state, $employeeId, $mfaOptions, $m
         $state['profileUrl'] = $this->profileUrl;
         $state['mfaOptions'] = $mfaOptions;
         $state['methodOptions'] = $methodOptions;
-        $state['template'] = 'review.php';
+        $state['template'] = 'review';
```

Comment:
If the above needs to change, possibly also change this to `self::REVIEW_PAGE`.

Reply:
No change needed as described above.